src/atoms/Blockie.tsx

```diff
@@ -0,0 +1,13 @@
+import makeBlockie from 'ethereum-blockies-base64';
+import type { FunctionComponent } from 'react';
+
+import type { ImageProps } from '.';
+import { Image } from '.';
+
+interface Props extends ImageProps {
```

Comment:
Can you export this (as `BlockieProps` or something), so it can be imported by other stuff?

Reply:
I would suggest relying on TS inference, and using `React.ComponentProps<typeof Blockie>` when necessary.

Reply:
@vilsbole Explicitly defining props is common (see Rebass for example) and makes it easier for implementing components to extend the props of a component.

Reply:
Yeah, it depends on whether we want to use the same convention that Rebass is.